src/pathfora.js

```diff
@@ -866,6 +895,53 @@
       return valid;
     },
 
+    entityFieldChecker: function (widget, fieldName, found) {
+      if (!found || !found.length) {
+        return true;
+      }
+
+      // for each template found...
+      for (var f = 0; f < found.length; f++) {
+        // parse the field name
+        var dataval = found[f].slice(2).slice(0, -2),
+            parts = dataval.split('|'),
+            def;
+
+        // get the default (fallback) value
+        if (parts.length > 1) {
+          def = parts[1].trim();
+        }
+
+        // check for subfields if the value is an object
+        var split = parts[0].trim().split('.');
```

Comment:
regex can also prevent the potential situation where a period was in the fallback and it was intended to be punctuation rather than dot notation. (e.g. if a period has no spaces before or after it, can be considered to be dot but if it has a space it could be something like `Dr.`